CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Changelog for dx-toolkit releases
+
+This change log spiritually adheres to [these guidelines](http://keepachangelog.com/en/0.3.0/).
+
+Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Security
+
+
+## [Unreleased]
```

Comment:
Is this line placeholder for the actual version number once we do the deployment?